scripts/zones/Nyzul_Isle/instances/nashmeiras_plea.lua

```diff
@@ -34,12 +34,13 @@ end
 function onInstanceProgressUpdate(instance, progress)
     if (progress == 4) then
         local chars = instance:getChars()
+        local entryPos = instance:getEntryPos()
 
+        DespawnMob(ID.mob[59].RAZFAHD, instance)
         for i, v in pairs(chars) do
             v:startEvent(203)
+            v:setPos(entryPos.x, entryPos.y, entryPos.z, entryPos.rot)
```

Comment:
Still have the `player:setPos(-444, -4, 420, 127)` down in the eventFinish~

Reply:
_Oof!_ Nice catch!